### PR TITLE
Update UG for constraints of name

### PR DESCRIPTION
**Pull request by teamotitan**
Previously, we did not mention if name will be case-sensitive or not

Let's
* Include the comment inside the UG constraints, so that users will be more informed

NOTE: We discussed that we should include it inside the edit command, but I decided not to, because under the constraints in edit command, we mentioned
* At least one of the optional fields must be provided. You may refer to
[Constraints of Add command](#constraints) for acceptable values of each field.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -139,7 +139,9 @@ Adds a student to the address book.
 
 ##### Constraints
 
-* **NAME** must only contain alphanumeric characters and spaces.
+* **NAME** must only contain alphanumeric characters and spaces. 
+* **NAME** is case-insensitive, i.e. you can use `jOhN Doe`
+  and we will help you to format it to become `John Doe`
 * **SCHEDULE** must be in the format of `DAY_OF_THE_WEEK`-`START_TIME`-`END_TIME`.
 * **DAY_OF_THE_WEEK** includes `Monday` `Tuesday` `Wednesday` `Thursday` `Friday` `Saturday` `Sunday`.
```
Comment (Feng1231) [inline docs/UserGuide.md:146]: can you help to indent line 146 and 147? I realised these two should be sub pointers for line 145. 
Also please help to change 146 to 
**DAY_OF_THE_WEEK** is one of `Monday` `Tuesday` `Wednesday` `Thursday` `Friday` `Saturday` `Sunday`.

**Review by Feng1231 — COMMENTED** [review]
Please help to indent 146 and 147 to make it clearer to user

**Review by Soumyadip-cmd — COMMENTED** [review]
Can help to check if DAY_OF_THE_WEEK is case insensitive. If it is, can help to add the note about it. Same for subject

**Comment by Feng1231** [comment]
Also, please ensure you add a space after the `*` in your future PR descriptions.
e.g. `*Include the comment ...` should be `* Include the comment ...`

**Comment by teamotitan** [comment]
Thanks guys for the input. I will make the necessary changes

**Review by Feng1231 — APPROVED** [review]

**Pull request merged**